lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -4,6 +4,19 @@
 
 module KubernetesDeploy
   class KubernetesResource
+    def self.logger=(value)
+      @logger = value
+    end
+
+    def self.logger
+      @logger ||= begin
+        l = Logger.new($stderr)
+        l.formatter = proc do |_severity, _datetime, _progname, msg|
+          "#{msg}\n"
```

Comment:
> why are there already double timestamps on that ERROR?

That's just someone's raw logs from `rake db:migrate`... I probably copied it from a deploy of a test app whose logging is messed up. Nothing to do with the deploy script.

So you'd want three loggers then? One that's genuinely raw for log dumping, one for kube status, and one for human-friendly messages? I'm still not convinced that it's useful for kube status, which is supposed to be machine-readible json + a tag to identify the lines. I guess you're asking me to justify "why not?" and the answer comes down to that I don't see a compelling "why?" here.